apps/meteor/client/views/hooks/useMemberExists.ts

```diff
@@ -0,0 +1,10 @@
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+
+export const useMemberExists = (roomId: string, username: string, override?: boolean) => {
```

Comment:
I think we should use an object for parameters here, so we can use named parameters instead of providing booleans to the function, it makes it harder to understand. What about a `{ ..., isMember: boolean }`?

Reply:
Yes. It makes sense.

Reply:
@MarcosSpessatto Should I also change parameters for useUserInfoActions to an object? It will make the code more understandable.